dominion/dominion_ui/src/components/Modal/GetLocation.js

```diff
@@ -43,7 +42,7 @@ class GetLocation extends React.Component {
           return addressObj.length > 0;
         });
         if (countryfound) {
-          const url = _.findIndex(countries, countryfound);
+          const url = countries.findIndex(countryfound);
```

Comment:
I think instead of looping `countries` twice, first for `find` and then for `findIndex`, we can combine them using `Object.entries`:
```javascript
const addressContains = (country) => {
  return addresses.find(address => address.long_name === country.name) !== undefined;
};
const foundEntry = Object.entries(countries).find(([slug, country]) => {
  return addressContains(country);
});
if (foundEntry) {
  const [url, ] = foundEntry;
}
```